qiskit_experiments/framework/experiment_data.py

```diff
@@ -1067,7 +1067,6 @@ def _add_result_data(self, result: Result, job_id: Optional[str] = None) -> None
                         # different parameter binds which is trivial because
                         # qiskit-experiments does not support parameter binding
                         # to pubs currently.
-                        joined_data = joined_data[0]
```

Comment:
Thanks, I did some more testing and confirmed this change is good. We need to remove the comment above it as well since it was explaining this line. This line came from the way I added level 1 support to `BackendSampler` for testing. I need to update that as well.

We can fix the comment here and merge or not. I will work on the tests in #1473 and will remove the comment there if it is not removed here.

Reply:
I have removed the comment :)